tests/reftest.rs

```diff
@@ -43,7 +43,8 @@ bitflags!(
 
 fn main() {
     let args: Vec<String> = env::args().collect();
-    let mut parts = args.tail().split(|e| &**e == "--");
+
+    let mut parts = args.split_first().unwrap().1.split(|e| &**e == "--");
```

Comment:
Nit: I think `args.split_first().unwrap().1` is better written as `args[1..]`. r=me with this.